python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -1113,6 +1113,7 @@ def measureDipoles(self, measRecord, exposure, posExp=None, negExp=None):
         ):
             measRecord.set(self.classificationFlagKey, False)
             measRecord.set(self.classificationAttemptedFlagKey, False)
+            # TODO: Do we want to fail here, or just leave it marked as not fit?
```

Comment:
@isullivan : this question still needs answering: I would lean towards not "failing" them, and just leaving them marked as not being attempted to be fit nor as dipoles.